auction9-frontend/src/app/data-table/data-table-details/data-table-details.component.html

```diff
@@ -33,13 +34,13 @@ <h4>Number of bids:
 		      +11
       </button>
 		</h4>
-
   	</div>
     <p>
       {{auction?.description}}
     </p>
   </mat-card-content>
   <mat-card-actions>
+    <mat-form-field class="input-bid">
```

Comment:
I'm not sure if this input is duplicate?

Reply:
Removed.